src/NuGet.Core/NuGet.Commands/CommandArgs/PackArgs.cs

```diff
@@ -22,6 +22,7 @@ public class PackArgs
         public ILogger Logger { get; set; }
         public LogLevel LogLevel { get; set; }
         public bool IncludeReferencedProjects { get; set; }
+        public bool InstallPackageToV3Feed { get; set; }
```

Comment:
@rrelyea how about InstallPackageToOutputPath ? I propose that the flag (which is a boolean, doesn't accept any value)does the following:
1) Installs the package to the output path as specified by user
2) Prevents the normal nupkg (and symbols nupkg) from being generated.
3) Logs where the package was installed

Reply:
Resolved. Updated name. Normal nupkg and symbols are no longer generated.